src/lib/containers/table/SynapseTable.tsx

```diff
@@ -393,6 +417,31 @@ export default class SynapseTable extends React.Component<
     )
   }
 
+
+  enableResize() {
+    if (!this.resizer) {
+      if (ReactDOM) {
+        const domNode: Text | Element | null = ReactDOM!.findDOMNode(this)
```

Comment:
when I pulled down the PR eslint gave a warning 'dont use findDOMNode' it looks like there is a more preferable method using [refs instead](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-find-dom-node.md)

Reply:
I'll try it out, thanks!